spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -705,6 +705,47 @@ describe("RustCrypto", () => {
         expect(resetKeyBackup.mock.calls).toHaveLength(2);
     });
 
+    it("bootstrapSecretStorage saves megolm backup key if already cached", async () => {
```

Comment:
Can we get an integ test for this as well, please? (One which works only through the public APIs of the js-sdk, together with fetchMock)